apps/web/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Label } from './Label';
+
+interface DropdownOptionProps {
+  id: string;
+  value: string;
+}
+interface DropdownProps {
+  dropdownOptions: DropdownOptionProps[];
+}
+
+export const Dropdown = ({ dropdownOptions }: DropdownProps) => {
```

Comment:
This entire component should be under `components/generic` which gets defaultValue, options, label and all those stuffs as props

Reply:
Moved to a generic component.